Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -275,22 +275,21 @@ function GetCodeFlowChanges
      {
         Write-Verbose "Change count: $($changes.Count)"
      }
-     
+
      return $changes
 }
 
-
 function TryGetCodeFlowChangeTrackingId
 {
-    param ($changes, [string]$path, [Ref][int]$changeId)
-    $change = $changes.ChangeEntries | Where-Object {$_.Modified.Path -eq $path}
+    param ($changes, $path, [Ref][int]$changeId)
 
+    $change = @($changes.ChangeEntries | Where-Object {$_.Modified.Path -eq $path})
```

Comment:
I am forcing an array here to overcome PS's inconsistent behaviour of conflating a single-element array with the element itself. I was getting errors in my test, but not in the product.